docs/using.rst

```diff
@@ -409,6 +409,13 @@ Certbot is working hard to improve the renewal process, and we
 apologize for any inconvenience you encounter in integrating these
 commands into your individual environment.
 
+.. note:: ``certbot renew`` exit status will only be 1 if renewal attempt failed.
+  This means ``certbot renew`` exit status will be 0 if no cert needs to be updated.
+  If you write a custom script and expect to run a command only after a cert was actually renewed
+  you will need to use the ``--renew-hook`` since the exit status will be 0 both on successful renewal
```

Comment:
I think we probably want to recommend `--post-hook` here. There are a couple differences:

1. `--renew-hook` will be run once for each cert is renewed. This means if `certbot renew` renews five certificates in one run, the `--renew-hook` is run five times as well.
2. `--renew-hook` is also saved for future renewals. If you renew a cert using `certbot renew --renew-hook "service nginx reload"`, future renewals can just use `certbot renew` and "service nginx reload" will be run whenever that cert is renewed.